libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectMapper.kt

```diff
@@ -57,7 +57,7 @@ internal typealias SDKTunnelInfo =
 internal typealias SDKRailwayCrossingInfo =
     com.mapbox.navigation.base.trip.model.roadobject.railwaycrossing.RailwayCrossingInfo
 
-internal fun com.mapbox.navigator.RoadObject.mapToRoadObject(): RoadObject {
+internal fun com.mapbox.navigator.RoadObject.mapToRoadObject(): RoadObject? {
```

Comment:
nullable result was a bad idea, it's better to filter out unsupported types based on `SUPPORTED_ROAD_OBJECTS`